includes/commands/class-mu-migration-base.php

```diff
@@ -37,8 +40,9 @@ protected function process_args( $default_args = array(), $args = array(), $defa
 	/**
 	 * Run through all posts and executes the provided callback for each post
 	 *
-	 * @param $query_args
-	 * @param $callback
+	 * @param array    $query_args
```

Comment:
`$query_args` is essentially everything that WP_Query accepts, we can just add a description saying that. If the parameter name is self-explanatory I'd rather keep it without description as the description would just say the obvious.